Cargo.lock

```diff
@@ -2666,6 +2666,7 @@ dependencies = [
  "nu-path",
  "nu-protocol",
  "nu-utils",
+ "strip-ansi-escapes",
```

Comment:
Unintentional? Is this what is breaking CI?

Reply:
I don't believe strip-ansi-escape uses chrono. Typically I checkin Cargo.lock files that are different than the repo. Is that not the right thing to do?

Reply:
You shouldn't check them in if they're unrelated to the change. It indicates something else is wrong with your environment, or something is wrong on master. But if something is wrong on master we should fix that in its own PR.